Common/Brokerages/KrakenBrokerageModel.cs

```diff
@@ -95,6 +95,20 @@ public override bool CanSubmitOrder(Security security, Order order, out Brokerag
             }
             return base.CanSubmitOrder(security, order, out message);
         }
+        
+        /// <summary>
+        /// Kraken does no support update of orders
+        /// </summary>
+        /// <param name="security"></param>
+        /// <param name="order"></param>
+        /// <param name="request"></param>
+        /// <param name="message"></param>
+        /// <returns></returns>
+        public override bool CanUpdateOrder(Security security, Order order, UpdateOrderRequest request, out BrokerageMessageEvent message)
```

Comment:
nit - missing method docs

Reply:
*needs a rebase too 👍

Reply:
Can you explain please what do you mean by 'rebase method docs'?